db/schema.rb

```diff
@@ -478,6 +478,7 @@
     t.boolean "enable_individual_requests", default: true, null: false
     t.boolean "enable_quantity_based_requests", default: true, null: false
     t.boolean "ytd_on_distribution_printout", default: true, null: false
+    t.boolean "use_single_step_invite_and_approve_partner_process", default: false
```

Comment:
Will add a rake task in a separate PR to backfill the current records with default `false`, so it's more uniform. Don't think it fits in this PR. And will add a `not NULL` constraint.